tox.ini

```diff
@@ -41,30 +41,38 @@ commands =
     # example tox run -e py39-profile-docs -- -o profile.svg -n 100
     profile-docs: python ./tools/profile.py {posargs}
 
-# tests can be ran with or without coverage (see examples below), 
 # it is recommended to run compile before running tests (see examples below), 
 # tox run -e compile,py39-tests 
 # if you want to skip the assets compilation step you can run the tests without 
 #`compile`, for example:
 # tox run -e py39-tests 
 # run tests with a specific Sphinx version
 # tox run -e compile,py39-sphinx61-tests
-# run tests without coverage
-# tox run -e compile,py39-tests-no-cov
-[testenv:py3{9,10,11,12}{,-sphinx61,-sphinxdev,}-tests{,-no-cov}]
-description = "Run tests Python and Sphinx versions. If a Sphinx version is specified, it will use that version vs the default in pyproject.toml"
+# tox run -e compile,py310-sphinxdev-tests
+# all possible combinations are listed below, and are based on the Python
+# version supported by the Sphinx version (expanding here because it is easier to read)
+#py3{9,10}-sphinx{61,62,70,71}-tests,
+#py3{10,11,12}-sphinx{latest,dev}-tests,
+#py3{9,10,11,12}-sphinx{72,73,74,80}-tests,
+[testenv:py3{9,10}-sphinx{61,62,70,71}-tests,py3{9,10,11,12}-sphinx{72,73,74,80}-tests,py3{10,11,12}-sphinx{latest,dev}-tests]
+description = "Run tests with a specific Sphinx version"
 # need to ensure the package is installed in editable mode
 package = editable
 extras =
-    test # install dependencies - defined in pyproject.toml
+    test # test dependencies - defined in pyproject.toml
 deps = 
     coverage[toml]
-    py39-sphinx61-tests: sphinx~=6.1.0
-    py312-sphinxdev: sphinx[test] @ git+https://github.com/sphinx-doc/sphinx.git@master
+    sphinx61: Sphinx>=6.1,<6.2
+    sphinx62: Sphinx>=6.2,<6.3
+    sphinx70: Sphinx>=7.0,<7.1
+    sphinx71: Sphinx>=7.1,<7.2
+    sphinx72: Sphinx>=7.2,<7.3
+    sphinx80: Sphinx>=8.0,<8.1
+    sphinxlatest: Sphinx
+    sphinxdev: sphinx[test] @ git+https://github.com/sphinx-doc/sphinx.git@master
```

Comment:
I erred on adding all versions here but since we are not running tests against all of them I can remove some of them.

Reply:
IMO the latest point-release of 6.x, 7.x, 8.x, plus current main is enough